apps/www/src/components/custom/HeroCards.tsx

```
@@ -0,0 +1,79 @@
+      properties: "239",
+      startingFrom: "$10",
+      city: "London, United Kingdom",
+      location: "United Kingdom",
+    },
+  ];
+
+  return (
+    <div className="flex flex-row flex-wrap justify-center gap-x-4 gap-y-8 md:gap-x-8 md:gap-y-28 ">
+      {/* // <div className="flex flex-row flex-wrap justify-center gap-8"> */}
+      {cardData.map((card, index) => (
+        <PinContainer title={card.city} href="#">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)